pkg/apis/core/v1/resource_test.go

```diff
@@ -13,6 +13,32 @@ import (
 )
 
 var _ = Describe("resource.Resource", func() {
+	Context("ResourceWithTags", func() {
+		var apiClient api.API
+		var rwt *ResourceWithTag
+
+		BeforeEach(func() {
+			if !isIntegrationTest {
+				Skip("integration build-flag not set")
+			}
+
+			a, err := api.NewAPI(api.WithClientOptions(client.AuthFromEnv(false)))
+			Expect(err).ToNot(HaveOccurred())
+			apiClient = a
+			rwt = &ResourceWithTag{Identifier: "94a4e6561ba944dfb9f5d2dfd7f10d78", Tag: "abc"}
```

Comment:
hm let's not add integration tests requiring a specific resource to exist in the Engine. In this case, I think testing the `EndpointURL` and `FilterRequestURL` methods is enough - each with all the operations `ResourceWithTag` supports